modules/nw-sriov-networknodepolicy-object.adoc

```diff
@@ -41,6 +41,8 @@ spec:
 <2> The namespace where the SR-IOV Network Operator is installed.
 
 <3> The resource name of the SR-IOV network device plugin. You can create multiple SR-IOV network node policies for a resource name.
++
+When specifying a name, be sure to not use special characters such as `-` in the `resourceName`.
```

Comment:
How about the correct resource name express is `^[a-zA-Z0-9_]+$`

Reply:
if the resource name contain '-', the webhook can blocked it and return error, eg

Error from server (resource name "e810-c" contains invalid characters, the accepted syntax of the regular expressions is: "^[a-zA-Z0-9_]+$"): error when creating "snnp.yaml": admission webhook "operator-webhook.sriovnetwork.openshift.io" denied the request: resource name "e810-c" contains invalid characters, the accepted syntax of the regular expressions is: "^[a-zA-Z0-9_]+$"
 
unless the customer disable the webhook feature.

Reply:
@zhaozhanqi I've updated based on your feedback. Does the update to the document work?